.gitlab-ci.yml

```diff
@@ -115,6 +114,22 @@ prepare:squidex:branch:
     - mkdir -p first_run
     - touch first_run/squidex_app_create
 
+sync:test:pr:
```

Comment:
is this a new job? why is it here if i may ask?

Reply:
Thanks! I've removed, we added this together as a test.